getting_started/HowTo.rst

```diff
@@ -0,0 +1,114 @@
+How to
+===============
+
+How to download the game
+************************
+
+To be able to download the game you have to be in the testing groups or have buy the game
+
+#. Download itch.io launcher at this link :  https://itch.io/app
+
+#. Once you have it install , Create a Itch account
```

Comment:
comma, colon and other signals are better and more correct when put just after the word:

```suggestion
#. Once you have it install, Create a Itch account
```

Before vowel (in general), uses **an** instead of **a**

```suggestion
#. Once you have it install, Create an Itch account
```

Verbal tense, and plain lowercase it visually better I think

```suggestion
#. Once you have it installed, create an Itch account
```